planner/solver.go

```diff
@@ -331,7 +332,9 @@ func (s *Solver) generateSolutions(ctx context.Context, req *PlanningReq, timeMa
 			return resp
 		}
 
-		iowrappers.Logger.Infof("number of places by price matcher is %d", len(placesByPrice))
+		iowrappers.Logger.Infof("Original of places is %d", len(places))
```

Comment:
Consider changing the log message to `Before filtering, the number of places is %d`.

Reply:
Done